docs/design/solutions/hub-and-spoke-tpe/flow_update_install.txt

```diff
@@ -0,0 +1,66 @@
+@startuml
+title FLOW update
+
+participant Client
+participant NorthBound as NB
+control kilda.northbound
+control kilda.flow
+participant "FLOW SplitterBolt" as SplitterBolt
+participant "FLOW CrudBolt" as CrudBolt
+participant "FLOW ErrorBolt" as ErrorBolt
+participant "FLOW NorthboundReplyBolt" as NorthboundReplyBolt
+participant FlowService
+participant DAO
+participant "FLOW TransactionBolt" as TransactionBolt
+participant "FLOW SpeakerBolt" as SpeakerBolt
+
+Client -> NB: /flows PUT
+NB -> kilda.flow: CommandMessage\nFlowUpdateRequest
+kilda.flow -> SplitterBolt: CommandMessage\nFlowUpdateRequest
+SplitterBolt -> CrudBolt: CommandMessage\nFlowUpdateRequest
+note right of CrudBolt: validate the flow (using FlowValidator)
+note right of CrudBolt: new path (using PathComputer.getPath)
+note right of CrudBolt: build flow pair (state=ALLOCATED)
+note right of CrudBolt: save flow (FlowCache.updateFlow)
+note right of CrudBolt: allocate cookie for flow
+note right of CrudBolt: allocate transit vlan (forward)
+note right of CrudBolt: allocate transit vlan (reverse)
+note right of CrudBolt: allocate meters
+note right of CrudBolt: new FlowInfoData
+loop in forward, reveres
+  opt
+    CrudBolt -> FlowService: update flow
```

Comment:
here is the same: you allocate new bunch of resources, but didn't release previously used